### PR TITLE
[SMALLFIX] Clean up temporary alluxio-master directory

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

File: minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java [hunk 1]
```diff
@@ -219,7 +219,7 @@ public FileSystem getClient() throws IOException {
   }
 
   private static String uniquePath() throws IOException {
-    return File.createTempFile("Alluxio", "").getAbsoluteFile() + "U" + System.nanoTime();
+    return AlluxioTestDirectory.createTemporaryDirectory("alluxio-master").getAbsolutePath();
```
Comment (yupeng9) [inline minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java]: Would there be collisions on this folder?

**Comment by aaudiber** [comment]
Jenkins, test this please

File: minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java (hunk 1 above)
Comment (aaudiber) [inline minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java]: the createTemporaryDirectory function appends a UUID to the name

Comment (yupeng9) [inline minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java]: sg

**Comment by yupeng9** [comment]
LGTM. Thanks for the fix

**Comment by aaudiber** [comment]
alluxio-bot, check this please

**Pull request merged**